Meshing application not compiling

Error in tetrahedra_reconnect_utility.h

applications/MeshingApplication/custom_utilities/tetrahedra_reconnect_utility.h:247:21: error: no match for ‘operator==’ (operand types are ‘Kratos::Node<3ul>::Pointer {aka std::intrusive_ptr<Kratos::Node<3ul> >}’ and ‘std::nullptr_t’)
             if ((v0 == nullptr) || (v1==nullptr) || (v2 == nullptr) || (v3 == nullptr)) 

@roigcarlo @RiccardoRossi @playpaolo @jcotela

--- a/applications/MeshingApplication/custom_utilities/tetrahedra_reconnect_utility.h
+++ b/applications/MeshingApplication/custom_utilities/tetrahedra_reconnect_utility.h
@@ -244,7 +244,8 @@ public:
             Node<3>::Pointer v1 = mrModelPart.pGetNode(t->vertexes[1]->getID());
             Node<3>::Pointer v2 = mrModelPart.pGetNode(t->vertexes[2]->getID());
             Node<3>::Pointer v3 = mrModelPart.pGetNode(t->vertexes[3]->getID());
-            if ((v0 == nullptr) || (v1==nullptr) || (v2 == nullptr) || (v3 == nullptr))
+            if ((v0.get() == nullptr) || (v1.get()==nullptr) || (v2.get() == nullptr) || (v3.get() == nullptr))
+            
             {
                 if (!messageShown)
                     std::cout << "Invalid vertex access " << t->getID() <<"\n";

